networking/networking-dashboards.adoc

```diff
@@ -21,3 +21,15 @@ You can view both general networking metrics as well as OVN-Kubernetes metrics f
 To view general networking metrics, select *Networking/Linux Subsystem Stats* from the *Dashboards* drop-down list. You can view the following networking metrics from the dashboard: *Network Utilisation*, *Network Saturation*, and *Network Errors*.
 
 To view OVN-Kubernetes metrics select *Networking/Infrastructure* from the *Dashboards* drop-down list. You can view the following OVN-Kuberenetes metrics: *Networking Configuration*, *TCP Latency Probes*, *Control Plane Resources*, and *Worker Resources*.
+
+[id="ingress-dashboards"]
+== Ingress Operator dashboard
+You can view networking metrics handled by the Ingress Operator from the dashboard. This includes things like the following:
+
+* Incoming and outgoing bandwidth
+* HTTP error rates
+* HTTP server response latency
+
+To view these Ingress metrics, select *Networking/Ingress* from the *Dashboards* drop-down list. You can view Ingress metrics for the following: *Top 10 Per Route*, *Top 10 Per Namespace*, and *Top 10 Per Shard*.
```

Comment:
Totally unimportant but maybe you could be more specific by describing what the "top 10 per" slices are called. Maybe "categories"?
```suggestion
To view these Ingress metrics, select *Networking/Ingress* from the *Dashboards* drop-down list. You can view Ingress metrics for the following categories: *Top 10 Per Route*, *Top 10 Per Namespace*, and *Top 10 Per Shard*.
```

Again, up to you

Reply:
yeah good point, they're kinda different from other dashboard entries...categories sounds good, thanks 😄